ncm-metaconfig/src/main/metaconfig/ganesha/pan/config_v2.pan

```diff
@@ -0,0 +1,13 @@
+unique template metaconfig/ganesha/config_v2;
+
+variable GANESHA_SERVICE ?= 'nfs-ganesha';
+variable CTDB_MANAGES_GANESHA ?= false;
```

Comment:
it could be generalised to `SOMETHING_ELSE_MANAGES_GANEHSA` (but with better name :smile: